### PR TITLE
added typography component

### PR DESCRIPTION
**Pull request by KoushikKintali**
typography

### CODE REVIEW

**Review by sanket4120 — APPROVED** [review]
LGTM

**Review by tanishk15sharma — APPROVED** [review]
fine to me

**Comment by jyothishp994** [comment]
Looks good to merge

File: components/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                        <p class="text-success">Success: Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sed, natus.</p>
+                        <p class="text-danger">Danger: Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sed, natus.</p>
+                        <p class="text-warning">Warning: Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sed, natus.</p>
+                        <p class="text-info">Info: Lorem ipsum dolor sit, amet consectetur adipisicing elit. Sed, natus.</p>
+                    </div>
+                </div>
+                <div class="code-block">
+                    <iframe
+                    src="https://carbon.now.sh/embed/bo54mQP5IxXdS0d1CXE9"
+                    style="width: 100%; height: 266px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (jyothishp994) [inline components/typography/typography.html:137]: I think rem is preferred to use rather than px with any size more than 10,

**Review by jyothishp994 — COMMENTED** [review]
Ver well indented code, but would suggest to use rem rather than px. Made use of var for color. That is good practise.

**Pull request merged**